subprojects/ide/src/test/groovy/org/gradle/plugins/ide/internal/tooling/eclipse/EclipseModelBuilderDependenciesTest.groovy

```diff
@@ -50,6 +49,9 @@ class EclipseModelBuilderDependenciesTest extends AbstractProjectBuilderSpec {
         child1 = ProjectBuilder.builder().withName("child1").withParent(project).build()
         child2 = ProjectBuilder.builder().withName("child2").withParent(project).build()
         child3 = ProjectBuilder.builder().withName("child3").withParent(project).build()
+        new File(project.projectDir,"settings.gradle") << """
+            rootProject.name = 'test'
```

Comment:
Shouldn't the `AbstractProjectBuilderSpec` take care of creating `settings.gradle` for the root project?

Reply:
I didn't do this because I'm not sure some projects don't actually create a settings file, or a settings.gradle.kts file, so I wanted to restrict the scope of the changes to failing tests.